tools/config-gen/params.yaml

```diff
@@ -15,16 +31,308 @@
+  default: "http1"
+
+- flag-name: "max-conns-per-host"
+  config-path: "gcs-connection.max-conns-per-host"
+  type: "int"
+  usage: >-
+    The max number of TCP connections allowed per server. This is effective when
+    client-protocol is set to 'http1'. The default value 0 indicates no limit on
+    TCP connections (limited by the machine specifications).
+  default: "0"
+  
+- flag-name: "max-idle-conns-per-host"
+  config-path: "gcs-connection.max-idle-conns-per-host"
+  type: "int"
+  usage: "The number of maximum idle connections allowed per server."
+  default: "100"
+
+- flag-name: "enable-nonexistent-type-cache"
+  config-path: "metadata-cache.enable-nonexistent-type-cache"
+  type: "bool"
+  usage: >-
+    Once set, if an inode is not found in GCS, a type cache entry with type
+    NonexistentType will be created. This also means new file/dir created might
+    not be seen. For example, if this flag is set, and metadata-cache-ttl-secs
+    is set, then if we create the same file/node in the meantime using the same
+    mount, since we are not refreshing the cache, it will still return nil.
+
+- flag-name: "stackdriver-export-interval"
+  config-path: "metrics.stackdriver-export-interval"
+  type: "duration"
+  usage: >-
+    Export metrics to stackdriver with this interval. The default value 0
+    indicates no exporting.
+  default: "0s"
+
+- flag-name: "experimental-opentelemetry-collector-address"
+  config-path: "monitoring.experimental-opentelemetry-collector-address"
+  type: "string"
+  usage: "Experimental: Export metrics to the OpenTelemetry collector at this address."
+  deprecated: true
+  deprecation-warning: "Experimental flag: could be dropped even in a minor release."
+
+- flag-name: "log-file"
+  config-path: "logging.file-path"
+  type: "resolvedPath"
+  usage: >-
+    The file for storing logs that can be parsed by fluentd. When not provided,
+    plain text logs are printed to stdout when Cloud Storage FUSE is run 
+    in the foreground, or to syslog when Cloud Storage FUSE is run in the 
+    background.
+
+- flag-name: "log-format"
+  config-path: "logging.format"
+  type: "string"
+  usage: "The format of the log file: 'text' or 'json'."
+  default: "json"
+
+- flag-name: "experimental-enable-json-read"
+  config-path: "gcs-connection.experimental-enable-json-read"
+  type: "bool"
+  usage: >-
+    By default, GCSFuse uses the GCS XML API to get and read objects. When this
+    flag is specified, GCSFuse uses the GCS JSON API instead."
+  deprecated: true
+  deprecation-warning: "Experimental flag: could be dropped even in a minor release."
+
+- flag-name: "debug_fuse_errors"
+  type: "bool"
   usage: "This flag is currently unused."
+  default: "true"
   deprecated: true
   deprecation-warning: "This flag is currently unused."
 
 - flag-name: "debug_fuse"
   type: "bool"
-  default: "true"
+  usage: "This flag is currently unused."
+  deprecated: true
+  deprecation-warning: "This flag is currently unused."
```

Comment:
Can we instead write "debug fuse logs are now controlled by log-severity flag, please use log-severity trace to view the logs." or something similar?

Same for debug_gcs flag